packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2381,7 +2415,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
    * Setup a listener for roap events emitted by the media sdk.
    */
   private mediaRoapEventsListener() {
-    this.mediaConnection.on(
+    this.mediaConnection?.on(
```

Comment:
Why do we need these operators here? Is there a scenario where the `mediaConnection` would not exist now?